packages/relative-values/src/components/View/ListView/index.tsx

```diff
@@ -37,33 +39,67 @@ export const ListView: FC<Props> = ({ rv }) => {
 
   return (
     <div>
-      <div>
-        <DropdownButton text="Rows">
+      <div className="mb-2">
+        <DropdownButton text="Table Settings">
           {() => <AxisMenu axis="rows" />}
         </DropdownButton>
       </div>
       <div
-        className="grid grid-cols-3 border-r border-b border-gray-200 w-max"
+        className="grid grid-cols-6 border-r border-b border-gray-200 w-max"
         style={{
           gridTemplateColumns:
-            "minmax(100px, min-content) 180px minmax(100px, min-content)",
+            "minmax(220px, min-content)  minmax(140px, min-content) minmax(100px, min-content) minmax(160px, min-content) minmax(160px, min-content) minmax(160px, min-content)",
         }}
       >
-        <div className="sticky bg-white top-0 left-0 z-20" />
+        <CellBox header>
+          <div className="p-1 pt-2 text-sm font-semibold text-slate-600">Name</div>
+        </CellBox>
+        <CellBox header>
+          <div className="p-1 pt-2 text-sm font-semibold text-slate-600">ID</div>
+        </CellBox>
+        <CellBox header>
+          <div className="p-1 pt-2 text-sm font-semibold text-slate-600">Cluster</div>
+        </CellBox>
+        <CellBox header>
+          <div className="p-1 pt-2 text-sm font-semibold text-slate-600">Average Value</div>
+        </CellBox>
+        <CellBox header>
+          <div className="p-1 pt-2 text-sm font-semibold text-slate-600">Average Uncertainty (db)</div>
+        </CellBox>
         <ColumnHeader
           selectedItem={selectedItem}
           setSelectedItem={setSelectedItem}
         />
-        <CellBox header>
-          <div className="p-1">ID</div>
-        </CellBox>
         {sortedItems.map((item) => (
           <Fragment key={item.id}>
             <Header item={item} />
-            <RelativeCell id1={item.id} id2={selectedItem.id} rv={rv} />
             <CellBox>
-              <div className="p-1 font-mono text-xs">{item.id}</div>
+              <div className="p-2 font-mono text-xs text-slate-600">
```

Comment:
Should I start using `slate` instead of `gray` everywhere else? Seems excessive to use both, they're pretty similar.

Reply:
Yea, good point. I think I prefer slate.